cli/doppel/bin/__init__.py

```diff
@@ -0,0 +1 @@
+from doppel.cli.bin import analyze
```

Comment:
can you please remove this file? This is not necessary, as the special `bin/` directory containers _executable_ files.